src/access/examples/ExampleAccessControlDelegate.cpp

```diff
@@ -125,6 +127,34 @@ class SubjectStorage
         return CHIP_ERROR_INVALID_ARGUMENT;
     }
 
+public:
+    static const chip::TLV::Tag kTagNode = chip::TLV::ContextTag(1);
+
+    CHIP_ERROR Serialize(chip::TLV::TLVWriter & writer)
+    {
+        chip::TLV::TLVType container;
+        ReturnErrorOnFailure(writer.StartContainer(chip::TLV::AnonymousTag, chip::TLV::TLVType::kTLVType_Structure, container));
+
+        ReturnErrorOnFailure(writer.Put(kTagNode, mNode));
+
+        return writer.EndContainer(container);
+    }
+
+    CHIP_ERROR Deserialize(chip::TLV::TLVReader & reader)
+    {
+        ReturnErrorOnFailure(reader.Next(chip::TLV::AnonymousTag));
```

Comment:
Why not use the signature of Next that takes both tag and type, instead of doing the separate type check?